wombag-search: Add w-search-open-in-wallabag

EFLS opened the pull request:
I'm really liking Wombag as a Wallabag interface, especially because it feels so much like elfeed. I really love how you've made the live search so easy to use because of the help buffer.

While playing around with Wombag, I came up with the following function that might be useful to other users as well. I like to highlight passages as I read them, so I often visit articles in the web-interface. This pull request provides a function and a key for that.

Original commit comment: New function in wombag-search to open entry at point in the Wallabag web interface, using wombag-browse-url-function.

eeljen committed:
wombag-search: Add w-search-open-in-wallabag

* wombag-search.el (w-search-open-in-wallabag, w-search-mode-map):
New function in wombag-search to open entry at point in the
Wallabag web interface, using wombag-browse-url-function.

karthink commented:
> I'm really liking Wombag as a Wallabag interface, especially because it feels so much like elfeed. I really love how you've made the live search so easy to use because of the help buffer.

Thanks! But the credit for the interface design goes to @skeeto (for Elfeed, obviously) and @chenyanming (who wrote wallabag.el).

> New function in wombag-search to open entry at point in the Wallabag web interface, using wombag-browse-url-function.

This is useful, could you also bind this in `wombag-show-mode-map`?

eeljen committed:
wombag-show: Add w-show-open-in-wallabag

* wombag-show.el (w-show-open-in-wallabag, w-show-mode-map): Add a
function to open the currently shown Wombag article in the
Wallabag web interface, similar to w-search-open-in-wallabag.

EFLS commented:
> This is useful, could you also bind this in `wombag-show-mode-map`?

Good idea! See the new commit. My Elisp skills are limited, but I think I managed by copying your code, more or less.

skeeto commented:
@karthink, I didn't know about this, thanks! Glad to see more interfaces like this.

karthink commented:
@EFLS: Merged, thanks for the PR!
